widget/entry.go

```diff
@@ -792,7 +790,7 @@ func (e *Entry) Undo() {
 	}
 	pos := modify.Position
 	if modify.Delete {
-		pos += len(modify.Text)
+		pos += len([]rune(modify.Text))
```

Comment:
Here and anywhere else you want the length of a string in runes, it should be more efficient to use `utf8.RuneCountInString(modify.Text)` which avoids an allocation